game/graphics/opengl_renderer/OpenGLRenderer.h

```diff
@@ -18,6 +18,7 @@ struct RenderOptions {
   bool playing_from_dump = false;
 
   bool save_screenshot = false;
+  bool screenshot_should_compress = false;
```

Comment:
might be a good idea to have this on by default - png compression is lossless so the only downside is it takes slightly longer, which isn't a big deal.